editor/plugins/tiles/tile_set_editor.cpp

```diff
@@ -997,7 +997,7 @@ void TileSourceInspectorPlugin::_confirm_change_id() {
 }
 
 bool TileSourceInspectorPlugin::can_handle(Object *p_object) {
-	return p_object->is_class("TileSetAtlasSourceProxyObject") || p_object->is_class("TileSetScenesCollectionProxyObject");
+	return p_object && (p_object->is_class("TileSetAtlasSourceProxyObject") || p_object->is_class("TileSetScenesCollectionProxyObject"));
```

Comment:
For some reason this crashes without null check. I did not bother investigating why, it just makes no sense. No other plugin has this problem.